[
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/266279684",
    "html_url": "https://github.com/jupyter/notebook/issues/1964#issuecomment-266279684",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1964",
    "id": 266279684,
    "node_id": "MDEyOklzc3VlQ29tbWVudDI2NjI3OTY4NA==",
    "user": {
      "login": "MattKleinsmith",
      "id": 8968171,
      "node_id": "MDQ6VXNlcjg5NjgxNzE=",
      "avatar_url": "https://avatars.githubusercontent.com/u/8968171?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/MattKleinsmith",
      "html_url": "https://github.com/MattKleinsmith",
      "followers_url": "https://api.github.com/users/MattKleinsmith/followers",
      "following_url": "https://api.github.com/users/MattKleinsmith/following{/other_user}",
      "gists_url": "https://api.github.com/users/MattKleinsmith/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/MattKleinsmith/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/MattKleinsmith/subscriptions",
      "organizations_url": "https://api.github.com/users/MattKleinsmith/orgs",
      "repos_url": "https://api.github.com/users/MattKleinsmith/repos",
      "events_url": "https://api.github.com/users/MattKleinsmith/events{/privacy}",
      "received_events_url": "https://api.github.com/users/MattKleinsmith/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-12-11T12:37:04Z",
    "updated_at": "2016-12-11T12:37:04Z",
    "author_association": "NONE",
    "body": "The errors go away if you rerun all the cells. Closing.",
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/266436566",
    "html_url": "https://github.com/jupyter/notebook/issues/1964#issuecomment-266436566",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1964",
    "id": 266436566,
    "node_id": "MDEyOklzc3VlQ29tbWVudDI2NjQzNjU2Ng==",
    "user": {
      "login": "takluyver",
      "id": 327925,
      "node_id": "MDQ6VXNlcjMyNzkyNQ==",
      "avatar_url": "https://avatars.githubusercontent.com/u/327925?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/takluyver",
      "html_url": "https://github.com/takluyver",
      "followers_url": "https://api.github.com/users/takluyver/followers",
      "following_url": "https://api.github.com/users/takluyver/following{/other_user}",
      "gists_url": "https://api.github.com/users/takluyver/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/takluyver/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/takluyver/subscriptions",
      "organizations_url": "https://api.github.com/users/takluyver/orgs",
      "repos_url": "https://api.github.com/users/takluyver/repos",
      "events_url": "https://api.github.com/users/takluyver/events{/privacy}",
      "received_events_url": "https://api.github.com/users/takluyver/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-12-12T13:55:55Z",
    "updated_at": "2016-12-12T13:55:55Z",
    "author_association": "MEMBER",
    "body": "This means that something has saved an invalid notebook at some point in the past. 'stream' outputs are not meant to have metadata. When you re-run the notebook, it deletes the saved outputs and recreates the correctly. So there is or was a bug somewhere. I don't think the Jupyter Notebook application has ever had that particular bug, so it's likely that someone saved the notebook with another application (like Nteract or Pycharm).",
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/266508761",
    "html_url": "https://github.com/jupyter/notebook/issues/1964#issuecomment-266508761",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1964",
    "id": 266508761,
    "node_id": "MDEyOklzc3VlQ29tbWVudDI2NjUwODc2MQ==",
    "user": {
      "login": "rgbkrk",
      "id": 836375,
      "node_id": "MDQ6VXNlcjgzNjM3NQ==",
      "avatar_url": "https://avatars.githubusercontent.com/u/836375?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/rgbkrk",
      "html_url": "https://github.com/rgbkrk",
      "followers_url": "https://api.github.com/users/rgbkrk/followers",
      "following_url": "https://api.github.com/users/rgbkrk/following{/other_user}",
      "gists_url": "https://api.github.com/users/rgbkrk/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/rgbkrk/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/rgbkrk/subscriptions",
      "organizations_url": "https://api.github.com/users/rgbkrk/orgs",
      "repos_url": "https://api.github.com/users/rgbkrk/repos",
      "events_url": "https://api.github.com/users/rgbkrk/events{/privacy}",
      "received_events_url": "https://api.github.com/users/rgbkrk/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-12-12T18:21:33Z",
    "updated_at": "2016-12-12T18:21:33Z",
    "author_association": "MEMBER",
    "body": "nteract has never had that bug explicitly, though we do fold in whatever comes in off the message spec, assuming the kernel reports valid fields.",
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/266509967",
    "html_url": "https://github.com/jupyter/notebook/issues/1964#issuecomment-266509967",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1964",
    "id": 266509967,
    "node_id": "MDEyOklzc3VlQ29tbWVudDI2NjUwOTk2Nw==",
    "user": {
      "login": "takluyver",
      "id": 327925,
      "node_id": "MDQ6VXNlcjMyNzkyNQ==",
      "avatar_url": "https://avatars.githubusercontent.com/u/327925?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/takluyver",
      "html_url": "https://github.com/takluyver",
      "followers_url": "https://api.github.com/users/takluyver/followers",
      "following_url": "https://api.github.com/users/takluyver/following{/other_user}",
      "gists_url": "https://api.github.com/users/takluyver/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/takluyver/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/takluyver/subscriptions",
      "organizations_url": "https://api.github.com/users/takluyver/orgs",
      "repos_url": "https://api.github.com/users/takluyver/repos",
      "events_url": "https://api.github.com/users/takluyver/events{/privacy}",
      "received_events_url": "https://api.github.com/users/takluyver/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-12-12T18:25:47Z",
    "updated_at": "2016-12-12T18:25:47Z",
    "author_association": "MEMBER",
    "body": "That's a good point, it could be a kernel getting it wrong.",
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/266511172",
    "html_url": "https://github.com/jupyter/notebook/issues/1964#issuecomment-266511172",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1964",
    "id": 266511172,
    "node_id": "MDEyOklzc3VlQ29tbWVudDI2NjUxMTE3Mg==",
    "user": {
      "login": "rgbkrk",
      "id": 836375,
      "node_id": "MDQ6VXNlcjgzNjM3NQ==",
      "avatar_url": "https://avatars.githubusercontent.com/u/836375?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/rgbkrk",
      "html_url": "https://github.com/rgbkrk",
      "followers_url": "https://api.github.com/users/rgbkrk/followers",
      "following_url": "https://api.github.com/users/rgbkrk/following{/other_user}",
      "gists_url": "https://api.github.com/users/rgbkrk/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/rgbkrk/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/rgbkrk/subscriptions",
      "organizations_url": "https://api.github.com/users/rgbkrk/orgs",
      "repos_url": "https://api.github.com/users/rgbkrk/repos",
      "events_url": "https://api.github.com/users/rgbkrk/events{/privacy}",
      "received_events_url": "https://api.github.com/users/rgbkrk/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-12-12T18:30:02Z",
    "updated_at": "2016-12-12T18:46:00Z",
    "author_association": "MEMBER",
    "body": "Based on the [other metadata](https://raw.githubusercontent.com/tensorflow/tensorflow/master/tensorflow/tools/docker/notebooks/1_hello_tensorflow.ipynb) (search for colab in the notebook json), it appears to be a notebook that was created internally at Google in colaboratory (the non-public version).",
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/266511520",
    "html_url": "https://github.com/jupyter/notebook/issues/1964#issuecomment-266511520",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1964",
    "id": 266511520,
    "node_id": "MDEyOklzc3VlQ29tbWVudDI2NjUxMTUyMA==",
    "user": {
      "login": "takluyver",
      "id": 327925,
      "node_id": "MDQ6VXNlcjMyNzkyNQ==",
      "avatar_url": "https://avatars.githubusercontent.com/u/327925?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/takluyver",
      "html_url": "https://github.com/takluyver",
      "followers_url": "https://api.github.com/users/takluyver/followers",
      "following_url": "https://api.github.com/users/takluyver/following{/other_user}",
      "gists_url": "https://api.github.com/users/takluyver/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/takluyver/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/takluyver/subscriptions",
      "organizations_url": "https://api.github.com/users/takluyver/orgs",
      "repos_url": "https://api.github.com/users/takluyver/repos",
      "events_url": "https://api.github.com/users/takluyver/events{/privacy}",
      "received_events_url": "https://api.github.com/users/takluyver/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-12-12T18:31:17Z",
    "updated_at": "2016-12-12T18:31:17Z",
    "author_association": "MEMBER",
    "body": "ah, good catch",
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/312534454",
    "html_url": "https://github.com/jupyter/notebook/issues/1964#issuecomment-312534454",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1964",
    "id": 312534454,
    "node_id": "MDEyOklzc3VlQ29tbWVudDMxMjUzNDQ1NA==",
    "user": {
      "login": "LuoHuifen",
      "id": 17880617,
      "node_id": "MDQ6VXNlcjE3ODgwNjE3",
      "avatar_url": "https://avatars.githubusercontent.com/u/17880617?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/LuoHuifen",
      "html_url": "https://github.com/LuoHuifen",
      "followers_url": "https://api.github.com/users/LuoHuifen/followers",
      "following_url": "https://api.github.com/users/LuoHuifen/following{/other_user}",
      "gists_url": "https://api.github.com/users/LuoHuifen/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/LuoHuifen/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/LuoHuifen/subscriptions",
      "organizations_url": "https://api.github.com/users/LuoHuifen/orgs",
      "repos_url": "https://api.github.com/users/LuoHuifen/repos",
      "events_url": "https://api.github.com/users/LuoHuifen/events{/privacy}",
      "received_events_url": "https://api.github.com/users/LuoHuifen/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2017-07-03T02:15:03Z",
    "updated_at": "2017-07-03T02:15:03Z",
    "author_association": "NONE",
    "body": "i solved the problem by adding \"## coding=utf-8\".",
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/312690547",
    "html_url": "https://github.com/jupyter/notebook/issues/1964#issuecomment-312690547",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1964",
    "id": 312690547,
    "node_id": "MDEyOklzc3VlQ29tbWVudDMxMjY5MDU0Nw==",
    "user": {
      "login": "rgbkrk",
      "id": 836375,
      "node_id": "MDQ6VXNlcjgzNjM3NQ==",
      "avatar_url": "https://avatars.githubusercontent.com/u/836375?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/rgbkrk",
      "html_url": "https://github.com/rgbkrk",
      "followers_url": "https://api.github.com/users/rgbkrk/followers",
      "following_url": "https://api.github.com/users/rgbkrk/following{/other_user}",
      "gists_url": "https://api.github.com/users/rgbkrk/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/rgbkrk/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/rgbkrk/subscriptions",
      "organizations_url": "https://api.github.com/users/rgbkrk/orgs",
      "repos_url": "https://api.github.com/users/rgbkrk/repos",
      "events_url": "https://api.github.com/users/rgbkrk/events{/privacy}",
      "received_events_url": "https://api.github.com/users/rgbkrk/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2017-07-03T16:38:57Z",
    "updated_at": "2017-07-03T16:38:57Z",
    "author_association": "MEMBER",
    "body": "/cc @craigcitro",
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/312692190",
    "html_url": "https://github.com/jupyter/notebook/issues/1964#issuecomment-312692190",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1964",
    "id": 312692190,
    "node_id": "MDEyOklzc3VlQ29tbWVudDMxMjY5MjE5MA==",
    "user": {
      "login": "craigcitro",
      "id": 468559,
      "node_id": "MDQ6VXNlcjQ2ODU1OQ==",
      "avatar_url": "https://avatars.githubusercontent.com/u/468559?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/craigcitro",
      "html_url": "https://github.com/craigcitro",
      "followers_url": "https://api.github.com/users/craigcitro/followers",
      "following_url": "https://api.github.com/users/craigcitro/following{/other_user}",
      "gists_url": "https://api.github.com/users/craigcitro/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/craigcitro/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/craigcitro/subscriptions",
      "organizations_url": "https://api.github.com/users/craigcitro/orgs",
      "repos_url": "https://api.github.com/users/craigcitro/repos",
      "events_url": "https://api.github.com/users/craigcitro/events{/privacy}",
      "received_events_url": "https://api.github.com/users/craigcitro/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2017-07-03T16:47:56Z",
    "updated_at": "2017-07-03T16:47:56Z",
    "author_association": "NONE",
    "body": "Yep, that looks like a colab notebook that hasn't been updated since I stopped us from saving `stream` metadata everywhere.\r\n\r\nI'll fix this upstream in the TF repo.",
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/312697102",
    "html_url": "https://github.com/jupyter/notebook/issues/1964#issuecomment-312697102",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1964",
    "id": 312697102,
    "node_id": "MDEyOklzc3VlQ29tbWVudDMxMjY5NzEwMg==",
    "user": {
      "login": "craigcitro",
      "id": 468559,
      "node_id": "MDQ6VXNlcjQ2ODU1OQ==",
      "avatar_url": "https://avatars.githubusercontent.com/u/468559?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/craigcitro",
      "html_url": "https://github.com/craigcitro",
      "followers_url": "https://api.github.com/users/craigcitro/followers",
      "following_url": "https://api.github.com/users/craigcitro/following{/other_user}",
      "gists_url": "https://api.github.com/users/craigcitro/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/craigcitro/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/craigcitro/subscriptions",
      "organizations_url": "https://api.github.com/users/craigcitro/orgs",
      "repos_url": "https://api.github.com/users/craigcitro/repos",
      "events_url": "https://api.github.com/users/craigcitro/events{/privacy}",
      "received_events_url": "https://api.github.com/users/craigcitro/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2017-07-03T17:14:40Z",
    "updated_at": "2017-07-03T17:14:40Z",
    "author_association": "NONE",
    "body": "Actually, @MatthewKleinsmith where are you loading these notebooks from?\r\n\r\nIt looks like these were already dropped in [this commit](https://github.com/tensorflow/tensorflow/commit/26a244c8f8c0352cc8f90e4e4af2dcc314279b93), and I can confirm that colab isn't re-introducing them.",
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/312717810",
    "html_url": "https://github.com/jupyter/notebook/issues/1964#issuecomment-312717810",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1964",
    "id": 312717810,
    "node_id": "MDEyOklzc3VlQ29tbWVudDMxMjcxNzgxMA==",
    "user": {
      "login": "rgbkrk",
      "id": 836375,
      "node_id": "MDQ6VXNlcjgzNjM3NQ==",
      "avatar_url": "https://avatars.githubusercontent.com/u/836375?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/rgbkrk",
      "html_url": "https://github.com/rgbkrk",
      "followers_url": "https://api.github.com/users/rgbkrk/followers",
      "following_url": "https://api.github.com/users/rgbkrk/following{/other_user}",
      "gists_url": "https://api.github.com/users/rgbkrk/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/rgbkrk/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/rgbkrk/subscriptions",
      "organizations_url": "https://api.github.com/users/rgbkrk/orgs",
      "repos_url": "https://api.github.com/users/rgbkrk/repos",
      "events_url": "https://api.github.com/users/rgbkrk/events{/privacy}",
      "received_events_url": "https://api.github.com/users/rgbkrk/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2017-07-03T19:20:13Z",
    "updated_at": "2017-07-03T19:20:13Z",
    "author_association": "MEMBER",
    "body": "I summoned you a little late Craig, figured I'd let you know though. :smile:",
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/312729474",
    "html_url": "https://github.com/jupyter/notebook/issues/1964#issuecomment-312729474",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1964",
    "id": 312729474,
    "node_id": "MDEyOklzc3VlQ29tbWVudDMxMjcyOTQ3NA==",
    "user": {
      "login": "craigcitro",
      "id": 468559,
      "node_id": "MDQ6VXNlcjQ2ODU1OQ==",
      "avatar_url": "https://avatars.githubusercontent.com/u/468559?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/craigcitro",
      "html_url": "https://github.com/craigcitro",
      "followers_url": "https://api.github.com/users/craigcitro/followers",
      "following_url": "https://api.github.com/users/craigcitro/following{/other_user}",
      "gists_url": "https://api.github.com/users/craigcitro/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/craigcitro/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/craigcitro/subscriptions",
      "organizations_url": "https://api.github.com/users/craigcitro/orgs",
      "repos_url": "https://api.github.com/users/craigcitro/repos",
      "events_url": "https://api.github.com/users/craigcitro/events{/privacy}",
      "received_events_url": "https://api.github.com/users/craigcitro/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2017-07-03T20:37:44Z",
    "updated_at": "2017-07-03T20:37:44Z",
    "author_association": "NONE",
    "body": "😀  thanks for letting me know! I started the convo with you that led to https://github.com/jupyter/nbformat/issues/84 after realizing that Colab was setting erroneous metadata here.\r\n\r\nThis one's probably OK to close.",
    "performed_via_github_app": null
  }
]
